packages/core/src/node-parser/base.ts

```diff
@@ -140,7 +140,7 @@ export abstract class MetadataAwareTextSplitter extends TextSplitter {
     return nodes.reduce<TextNode[]>((allNodes, node) => {
       const metadataStr = this.getMetadataString(node);
       const splits = this.splitTextMetadataAware(
-        node.getContent(MetadataMode.ALL),
+        node.getContent(MetadataMode.NONE),
```

Comment:
You are right. Nice catch

https://github.com/run-llama/llama_index/blob/769e76a28aa497ec9885975a09eb8eec24713597/llama-index-legacy/llama_index/legacy/node_parser/interface.py#L175